lmdeploy/lite/apis/auto_awq.py

```diff
@@ -30,29 +29,20 @@
 
 def auto_awq(model: str,
              work_dir: str,
+             calib_dataset: str = 'c4',
```

Comment:
the cli is not updated. 
https://github.com/InternLM/lmdeploy/blob/68fa4b9a6566059c6ead5ec46d3314cfd73ca18f/lmdeploy/cli/lite.py#L7

Reply:
Done